.github/workflows/release-21_extrinsic-ordering-check-from-two.yml

```diff
@@ -0,0 +1,124 @@
+# This workflow performs the Extrinsic Ordering Check on demand using two reference binaries
+
+name: Release - Extrinsic API Check with reference bins
+on:
+  workflow_dispatch:
+    inputs:
+      reference_binary_url:
+        description: A url to a Linux binary for the node containing the reference runtime to test against
+        default: https://releases.parity.io/cumulus/v0.9.230/polkadot-parachain
+        required: true
+      binary_url:
+        description: A url to a Linux binary for the node containing the runtime to test
+        default: https://releases.parity.io/cumulus/v0.9.270-rc7/polkadot-parachain
+        required: true
+
+jobs:
+  check:
+    name: Run check
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    env:
+      REF_URL: ${{github.event.inputs.reference_binary_url}}
+      BIN_REF: polkadot-parachain-ref
+      BIN_URL: ${{github.event.inputs.binary_url}}
+      BIN_BASE: polkadot-parachain
+      TMP: ./tmp
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - category: assets
```

Comment:
the category key doesn't seem to be used anywhere

Reply:
That could be removed indeed, it comes from the other workflow and documents the sub-folder where this runtime comes from. So besides this "documentation" there is no need today and I don't even really see a need later.